### ISSUE TITLE
Дизайн Кухни Кейт

### ISSUE THREAD

**Issue by dakand** [atls/design#94]
Вводные по проекту в этой [задаче](https://github.com/atls/design/issues/93).

Кратко о сути. Нужно сделать каталог для студии кухонь. Есть единственный референс по интерфейсу, который нравится заказчику https://dizant.com/. Нам необходимо создать подобный интерфейс, но со своей айдентикой. На полноценный дизайн-процесс мы здесь не рассчитываем, а делаем некоторые вводные для дальнейшей работы фронта.

План действий такой:
- [ ] Выбрать референсы из разряда #aesthetics
- [ ] Выбрать шрифт (только фришные)
- [ ] Выбрать цветовую палитру
- [ ] Сделать баннер, который мы сможем множить для социалок
- [ ] Спроектировать главную в десктопе и мобилке
- [ ] Спроектировать "Проекты и цены" в десктопе и мобилке
- [ ] Сделать базовый UI-кит: цвета, шрифты, пояснения к отступам сетки или правила расположения элементов (если таковые можно сформулировать.

Все это нужно делать крайне быстро. Нужно постараться закончить за пару дней. @bbbaso так что пока задача по анимации атлантиса на второй план.

**Comment by bbbaso** [comment]
@dakand предполагаю, что тут лучше юзать светлый бэкграунд и черный (темный) гротеск. По простой причине — она продает дизайн кухни, обоев и тд  (смежная тема с дизайном интерьера), и тут важно держать фокус на контенте, а белый бг и сдержанная типографика в этом сильно поможет, то есть я бы не смотрел в сторону широкой палитры и антиквы (+от самих фото будет куча цвета)

1. Собрал мудборд (частично там лежат рефы)
2. Фришная типографика будет наверн одна из дефолтных:
- Inter
- Roboto
- Manrope
_*если у нас уже есть Suisse, он бы зашел идеально_
3. Палитра в [файле](https://www.figma.com/file/bAQGD5onjiwtcfkRdf6Dkf/Kate-Studio?node-id=0%3A1)

_P.s. Учитывал момент со сроками, поэтому и думаю что этот вариант менее затратный: а) его нетяжело проектировать и рисовать, б) выглядит всегда эстетично (особенно если у нее есть хайрез фотки)_

**Comment by bbbaso** [comment]
@dakand Думаю вполне

**Comment by taqyon** [comment]
Ребята, она мне еще обещала сегодня дополнительных референсов скинуть, они вам нужны?

**Comment by dakand** [comment]
> Ребята, она мне еще обещала сегодня дополнительных референсов скинуть, они вам нужны?

Ну если они есть у неё, то нам нужно их увидеть так или иначе. Пусть только не сильно поздно.

**Comment by taqyon** [comment]
> > Ребята, она мне еще обещала сегодня дополнительных референсов скинуть, они вам нужны?
> 
> Ну если они есть у неё, то нам нужно их увидеть так или иначе. Пусть только не сильно поздно.

У нас уговор с ней на 16:00

**Comment by taqyon** [comment]
> > Ребята, она мне еще обещала сегодня дополнительных референсов скинуть, они вам нужны?
> 
> Ну если они есть у неё, то нам нужно их увидеть так или иначе. Пусть только не сильно поздно.

Она написала мне:

>Katerina, [10.10.2022 15:40]
>Иван, добрый день

>Katerina, [10.10.2022 15:40]
>У меня сейчас клиенты пришли, я ближе к 20:00 освобожусь и пришлю д/з

это будет проблемой?

**Comment by dakand** [comment]
> > > Ребята, она мне еще обещала сегодня дополнительных референсов скинуть, они вам нужны?
> > 
> > 
> > Ну если они есть у неё, то нам нужно их увидеть так или иначе. Пусть только не сильно поздно.
> 
> Она написала мне:
> 
> > Katerina, [10.10.2022 15:40]
> > Иван, добрый день
> 
> > Katerina, [10.10.2022 15:40]
> > У меня сейчас клиенты пришли, я ближе к 20:00 освобожусь и пришлю д/з
> 
> это будет проблемой?

да не, просто позже начнем

**Comment by bbbaso** [comment]
@dakand давай завтра утром созвонимся на счет структуры главной, потому что тут наверн нужно будет правильное позиционирование выбрать, я его пока плохо представляю, т.к. у нее есть и Кухни, и Обои, и Фрески и Дизайн квартиры под ключ. С остальными страницами пока все достаточно понятно. 

_P.s. Если есть коммент на этот счет, то можешь тут отписать. Я бы вечером прикинул что-ниб._

**Comment by taqyon** [comment]
> нужно будет правильное позиционирование выбрать, я его пока плохо представляю, т.к. у нее есть и Кухни, и Обои, и Фрески и Дизайн квартиры под ключ.

Абсолютно согласен. От себя добавлю, что она занимается не фабричными решениями и получается что каждое её изделие (результат труда) — это кейс, дело или проект. Это одна из причин, по которой ей нравится как реализован раздел проекты у её конкурента (референса).

**Comment by taqyon** [comment]
Екатерина прислала референсы. 

## Референсы

### 1. https://www.marya.ru/catalog/kuhni/

>нравится функциональное меню, что при наведении, открывается подменю: современный стиль, неоклассический стиль и т.д


>у них стиль белый с красным, поэтому и оформление сайта такое же

>когда мы подберем новые цвета для меня, так же можем сделать))

### 2. http://hyggepaint.ru/

>у этого сайта мне нравится цветовая гамма

### 3. 	https://zovrus.ru/

>чёрный фон на главной странице мне нравится) стильно смотрится, а дальше белый фон

**Comment by taqyon** [comment]
У меня вопрос: У неё есть фотки работ, но они всратные.

<details>
<summary>Пример</summary>

![photo_2022-10-10_20-01-38](https://user-images.githubusercontent.com/14002721/194923261-829378b7-6d45-4c7c-a142-29e03a4dd3c1.jpg)

</details>

Так вот, она хочет найти фотографа для того что бы тот отснял нормальный материал. Мне её не останавливать?

**Comment by TorinAsakura** [comment]
> **У меня вопрос:** У неё есть фотки работ, но они всратые.
> 
> Пример
> ![photo_2022-10-10_20-01-38](https://user-images.githubusercontent.com/14002721/194923261-829378b7-6d45-4c7c-a142-29e03a4dd3c1.jpg)
> 
> Так вот, она хочет найти фотографа для того что бы тот отснял нормальный материал. Мне её не останавливать?

Нет, но, нужна контрольная группа фотографий, и нужно, чтобы фотограф был с опытом, и не свадебных съёмок, я могу это на себя взять, у меня уже был аналогичный опыт

**Comment by taqyon** [comment]
> Нет, но, нужна контрольная группа фотографий, и нужно, чтобы фотограф был с опытом, и не свадебных съёмок, я могу это на себя взять, у меня уже был аналогичный опыт

Да, возьми на себя пожалуйста, потому что ты с ней уже общался ранее, плюс вы вроде как рядом как я понял. Потому что контент здесь ядерный материал

**Comment by bbbaso** [comment]
@TorinAsakura @taqyon мысли в слух на счет фото:

На сколько я знаю и вижу в инете, каталоги кухни все в виде рендеров, полагаю раз они сами дизайнят, то и у них они должны быть. Реальные фотки были б нужны скорее для раздела "Примеры работ" или "Отзывы"

И помогут наверн фотки уже каталога обоев/фресок, шоурумов и самого производства, чтобы как раз сделать акцент на "нефабричности". Мб еще фотки команды хорошо отразят момент собственного производства. (можно будет сделать блок на 1.5-2 экрана на странице "производство")

**Comment by TorinAsakura** [comment]
> @TorinAsakura @taqyon мысли в слух на счет фото:
> 
> На сколько я знаю и вижу в инете, каталоги кухни все в виде рендеров, полагаю раз они сами дизайнят, то и у них они должны быть. Реальные фотки были б нужны скорее для раздела "Примеры работ" или "Отзывы"
> 
> И помогут наверн фотки уже каталога обоев/фресок, шоурумов и самого производства, чтобы как раз сделать акцент на "нефабричности". Мб еще фотки команды хорошо отразят момент собственного производства. (можно будет сделать блок на 1.5-2 экрана на странице "производство")

Фотки нужны как раз для фолио, для раздела работ, это контент и он нужен.

**Comment by dakand** [comment]
@TorinAsakura посмотри промежуточный [результат](https://www.figma.com/file/bAQGD5onjiwtcfkRdf6Dkf/Kate-Studio?node-id=7%3A135) того, что сделал Рома. Мне нравится. Мы планируем, что к этому будет ещеё прилагаться базовый UI-кит и адаптив макетов. Поэтому подумай, что ещё может потребоваться для разработки.

**Comment by taqyon** [comment]
@bbbaso @dakand в общем на материалы потребуется время, потому что она немного медленная и непонятно есть ли эти материалы вообще в электронном виде. Она мне присылает фотки (с телефона) реальных материалов :)

**Comment by bbbaso** [comment]
> в общем на материалы потребуется время, потому что она немного медленная и непонятно есть ли эти материалы вообще в электронном виде. Она мне присылает фотки (с телефона) реальных материалов :)

Я в целом учитываю формат этих "материалов", стараюсь сильно не привязывать дизайн к контенту, хотя из него по сути сайт и будет. Если что, уменьшим фотки, но в ее интересах крупные фотки на сайте иметь, т.к. фотки на мыльцу можно и в ее инсте смотреть

**Comment by taqyon** [comment]
> > в общем на материалы потребуется время, потому что она немного медленная и непонятно есть ли эти материалы вообще в электронном виде. Она мне присылает фотки (с телефона) реальных материалов :)
> 
> Я в целом учитываю формат этих "материалов", стараюсь сильно не привязывать дизайн к контенту, хотя из него по сути сайт и будет. Если что, уменьшим фотки, но в ее интересах крупные фотки на сайте иметь, т.к. фотки на мыльцу можно и в ее инсте смотреть

Вот как пример обоев и того что она мне прислала: 

<details>
<summary>Пример</summary>

![photo_2022-10-11_21-10-02](https://user-images.githubusercontent.com/14002721/195328524-d96907be-1f0d-4a5e-916d-58cefa0203f8.jpg)


</details>

И она же сделала запрос на завод-производитель: 

>Иван, добрый день) жду от производителя исходники

**Comment by bbbaso** [comment]
@dakand 

Главная и каталог готовы (концепт)
Добавил одну фичу в каталог, коммент внуттри оставил по ней

[Ссылка на файл](https://www.figma.com/file/bAQGD5onjiwtcfkRdf6Dkf/Kate-Studio?node-id=32%3A105)

**Comment by TorinAsakura** [comment]
@dakand @bbbaso по структуре и оттенкам я видел себе это как-то так https://yootheme.com/wordpress-themes/juno

Ткнитесь и по итогу поделитесь пожалуйста впечатлениями. Да-да/нет-нет

**Comment by bbbaso** [comment]
> Ткнитесь и по итогу поделитесь пожалуйста впечатлениями. Да-да/нет-нет

Структура страниц выглядит оч шаблонной, хз когда последний раз такое встречал
Палитра старого формата, то есть эти оттенки не юзают сегодня + я эту палитру вижу в детском магазине разве что

_P.s. Свою палитру подбирал из предпочтений заказчика (бежевый, темный) + изначально старался не делать ее широкой (вот [тут](https://github.com/atls/design/issues/94#issuecomment-1273018296) писал почему)_

**Comment by TorinAsakura** [comment]
@bbbaso Добро. Нужно было перекрёстное мнение.

**Comment by TorinAsakura** [comment]
@dakand Ну, у меня в целом нареканий нет, нужно теперь с этим сходить к Екатерине и если ей норм - буду брать в работу.

**Comment by dakand** [comment]
@bbbaso ты сегодня будешь адаптивом заниматься?

**Comment by bbbaso** [comment]
@dakand 

Адаптивы готовы

[Ссылка на файл](https://www.figma.com/file/bAQGD5onjiwtcfkRdf6Dkf/Kate-Studio?node-id=122%3A538)

**Comment by taqyon** [comment]
Я закинул ей задание: 

1. Подготовить типы кухонь (реквесты клиентов)
2. Блок «Как мы работаем»

Займется сегодня, результат будет завтра вероятно
